packages/base/src/MaterialComponent.ts

```diff
@@ -126,6 +126,24 @@ export abstract class MaterialComponent<
     }
   }
 
+  protected proxyEventHandler(eventName: string, e: Event, options?: any) {
```

Comment:
Instead of `eventName: String` rather use `eventName: keyof MaterialComponentProps<PropType>`

Reply:
TIL

Reply:
done